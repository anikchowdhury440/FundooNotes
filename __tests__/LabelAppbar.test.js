import React from 'react';
import {shallow} from 'enzyme';
import LabelAppbar from '../src/components/dashboard/LabelAppbar';
import configureStore from 'redux-mock-store';
import thunk from 'redux-thunk'

const middlewares = [thunk]
const mockStore = configureStore(middlewares);
const initialState = {
    userId : '',
    userLabel : [],
    screenName : '',
    labelKey : ''
}
const store = mockStore(initialState)

describe('test LabelAppbar', () => {
    it('should match to snapshot', () => {
        const component = shallow(<LabelAppbar store = {store}/>)
        expect(component).toMatchSnapshot();
    })
})