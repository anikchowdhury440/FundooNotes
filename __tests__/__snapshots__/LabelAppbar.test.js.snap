// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test LabelAppbar should match to snapshot 1`] = `ShallowWrapper {}`;
